dnf/automatic/main.py

```diff
@@ -226,6 +226,7 @@ def main(args):
         with dnf.Base() as base:
             cli = dnf.cli.Cli(base)
             cli._read_conf_file()
+            cli.demands.resolving = True
```

Comment:
Please could you add comment in the code including bugzilla reference? Because dnfautomatic does not require the demand someone later could remove unneeded lines.

Reply:
@j-mracek done.